mysql/mysql.lua

```diff
@@ -0,0 +1,63 @@
+--[[
+# MySQL connector
+
+## Requirements:
+
+  * lua-cjson
+  * [luasql-mysql](https://keplerproject.github.io/luasql/index.html)
+
+  ```
+  sudo luarocks install luasql-mysql
+  sudo luarocks install lua-cjson
+  ```
+
+## Input parameters
+
+  * database - name of database to connect
+  * username - username to MySQL server
+  * password - password for username
+  * host - host of MySQL (default: 'localhost')
+  * port - port of MySQL (default: 3306)
+  * sqlQuery - SQL query to execute, e.g. `SELECT name, email from people`
+
+
+## Output parameters
+
+  Example response:
+
+  ```
+  {"name":"Jose das Couves","email":"jose@couves.com"}
+  {"name":"Manoel Joaquim","email":"manoel.joaquim@cafundo.com"}
+  {"name":"Maria das Dores","email":"maria@dores.com"}
```

Comment:
The response is wrong, there is a JSON line per each fetched item, but the connector should return ONE JSON string which will be decoded by pmio engine and the data could be extracted by curly braces notation.
My suggestion is to use 'response' array which will contain response objects as elements of the array.